pkg/vsphere/extraconfig/keys.go

```diff
@@ -161,7 +161,7 @@ func calculateScopeFromKey(key string) []string {
 	return scopes
 }
 
-func calculateKeyFromField(field reflect.StructField, prefix string, depth recursion) (string, recursion) {
+func calculateKeyFromField(log logrus.FieldLogger, field reflect.StructField, prefix string, depth recursion) (string, recursion) {
```

Comment:
I think changes like moving to trace.Operation should be separate from this PR, and part of a bigger move towards using context's in our APIs. I would prefer if you would stick with one logger for now, especially since we're not leveraging the two loggers differently anyway. Thanks for putting this fix up so quickly.